todo/src/style/index.css

```
@@ -0,0 +1,36 @@
+.bg-High {
```

why do you use upper case in naming ?

---

I change selector names.
```suggestion
.bg-high {
        background-color: var(--bg-light-high-priority);
    }
```